lib/publiccloud/utils.pm

```diff
@@ -214,17 +216,18 @@ sub define_secret_variable {
 # Get credentials from the Public Cloud micro service, which requires user
 # and password. The resulting json will be stored in a file.
 sub get_credentials {
-    my ($output_file) = @_;
+    my ($output_json) = @_;
     my $url = get_required_var('PUBLIC_CLOUD_CREDENTIALS_URL');
     my $user = get_required_var('_SECRET_PUBLIC_CLOUD_CREDENTIALS_USER');
     my $pwd = get_required_var('_SECRET_PUBLIC_CLOUD_CREDENTIALS_PASS');
-    $output_file //= '/root/credentials.json';
```

Comment:
I think this file is still needed in https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/data/publiccloud/terraform/gce.tf#L19